client/src/components/BizCard/BizCard.jsx

```diff
@@ -56,6 +58,7 @@ export default function BizCard({ props }) {
   const [pickDate, setPickDate] = useState(false);
   const [pickFuture, setPickFuture] = useState(false);
   const [userReviews, setUserReviews] = useState([]);
+  const [hoursBooked, setHoursBooked] = useState()
```

Comment:
i've read you should always put in an initial state for useState, but I'm not sure why.